client/app/core/config.js

```diff
@@ -1,20 +1,25 @@
 /* eslint angular/window-service: "off" */
-
+import { RootReducer } from '../reducers'
 var DEVEL_DOMAINS = [
   'localhost',
   '127.0.0.1',
   '[::1]'
 ]
 
 var isDevel = window._.includes(DEVEL_DOMAINS, window.location.hostname)
-
+const hasDevTools = angular.isDefined(window.__REDUX_DEVTOOLS_EXTENSION__)
 /** @ngInject */
-export function configure ($logProvider, $compileProvider, $qProvider) {
+export function configure ($logProvider, $compileProvider, $qProvider, $ngReduxProvider) {
   $logProvider.debugEnabled(isDevel)
   $compileProvider.debugInfoEnabled(isDevel)
 
   // TODO: Remove following line as per: https://docs.angularjs.org/guide/migration#migrate1.5to1.6-ng-services-$compile
   $compileProvider.preAssignBindingsEnabled(true)
 
   $qProvider.errorOnUnhandledRejections(false)
+  const storeEnhancers = []
+  if (hasDevTools) {
+    storeEnhancers.push(window.__REDUX_DEVTOOLS_EXTENSION__())
+  }
+  $ngReduxProvider.createStoreWith(RootReducer, [], storeEnhancers)
```

Comment:
I see you're not going with any Redux middlewares for now, which is fine :smiley: you can add them later on, once you need to handle async operations or side effects.

At first, Redux might feel too boilerplate'ish, but once you get to the point when you really need to handle async stuff (like HTTP) or side effects (like local storage) and make those part of logical flows in your Redux architecture, you'll see the true value of Redux :yum: